app/components/embed/companion_windows_component.html.erb

```diff
@@ -37,20 +37,13 @@
           <dl data-controller="iiif-metadata" class="metadataList left-drawer-content" data-action="iiif-manifest-received@window->iiif-metadata#drawMetadata"></dl>
         </section>
 
-        <section id="media-content" role="tabpanel" hidden>
-          <div class="header-background">
-            <h3>Media content</h3>
-          </div>
-          <ul class="left-drawer-content" data-controller="content-list" data-action="thumbnails-found@window->content-list#draw" role="tablist"></ul>
+        <section id="object-content" role="tabpanel" hidden>
+          <%= render ContentListComponent.new(viewer:, heading: viewer.content_heading) %>
         </section>
 
         <%= drawer_content %>
 
         <section id="rights" role="tabpanel" hidden>
-          <div class="header-background">
-            <h3>Rights</h3>
-          </div>
-
           <%= render RightsComponent.new(viewer:) %>
         </section>
```

Comment:
pulled the heading into the existing `RightsComponent` to match what is done in the new `ContentList` component (which needs to have the heading vary by content type)